From d3f4d2be1c9e8c21cdc7cdc9e4c7c345258d9f1b Mon Sep 17 00:00:00 2001
From: NeilBrown <neilb@suse.de>
Date: Wed, 29 Apr 2009 11:21:08 +1000
Subject: [PATCH 2/2] Makefile: use $(CC) more consistently.

Explicitly calling 'gcc' in some rules makes it hard to test with
other compilers.

Signed-off-by: martin f. krafft <madduck@debian.org>
---
 Makefile |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 3fb9c78..b89cd6f 100644
--- a/Makefile
+++ b/Makefile
@@ -111,13 +111,13 @@ mdadm.tcc : $(SRCS) mdadm.h
 
 mdadm.klibc : $(SRCS) mdadm.h
 	rm -f $(OBJS) 
-	gcc -nostdinc -iwithprefix include -I$(KLIBC)/klibc/include -I$(KLIBC)/linux/include -I$(KLIBC)/klibc/arch/i386/include -I$(KLIBC)/klibc/include/bits32 $(CFLAGS) $(SRCS)
+	$(CC) -nostdinc -iwithprefix include -I$(KLIBC)/klibc/include -I$(KLIBC)/linux/include -I$(KLIBC)/klibc/arch/i386/include -I$(KLIBC)/klibc/include/bits32 $(CFLAGS) $(SRCS)
 
 mdadm.Os : $(SRCS) mdadm.h
 	$(CC) -o mdadm.Os $(CFLAGS)  -DHAVE_STDINT_H -Os $(SRCS)
 
 mdadm.O2 : $(SRCS) mdadm.h
-	gcc -o mdadm.O2 $(CFLAGS)  -DHAVE_STDINT_H -O2 $(SRCS)
+	$(CC) -o mdadm.O2 $(CFLAGS)  -DHAVE_STDINT_H -O2 $(SRCS)
 
 test_stripe : restripe.c mdadm.h
 	$(CC) $(CXFLAGS) $(LDFLAGS) -o test_stripe -DMAIN restripe.c
-- 
1.6.2.4

